### PR TITLE
Andb parser int

### PR DESCRIPTION
**Pull request by pkommoju**
ANDB parser integration into native code.

Please read qpmodel/docs/using_andb_parser.txt before attempting to build it on Windows. I have not built native code on Linux, therefore I am not sure how the CMake system may have to be adapted to produce the effects of the script the document refers to.
Also to note is the fact that server/experimental/src/common has a public domain getopt header and C file to implement the option processing required for andb.exe (the command line driver for the native code). getopt.cpp needs to be excluded from SRC_LIST in server/experimental/src/common/CMakeLists.txt when running on Linux.

### CODE REVIEW

File: server/experimental/src/common/getopt.cpp [hunk 1]
```diff
@@ -0,0 +1,188 @@
+/*****************************************************************************
```
Comment (zhouqingqing) [inline server/experimental/src/common/getopt.cpp:1]: gflags lib already have function like/better than this - let's try to use gflags.

Comment (pkommoju) [reply server/experimental/src/common/getopt.cpp:1]: I will look into it.

File: server/hyrise-includes/ColumnType.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#ifndef __COLUMNTYPE_H_
```
Comment (zhouqingqing) [inline server/hyrise-includes/ColumnType.h:1]: let's try to remove other dependencies on hyrise code except the parser syntax file part

Comment (pkommoju) [reply server/hyrise-includes/ColumnType.h:1]: I will remove them.

Comment (pkommoju) [reply server/hyrise-includes/ColumnType.h]: done

File: server/experimental/src/common/getopt.cpp (hunk 1 above)
Comment (pkommoju) [reply server/experimental/src/common/getopt.cpp:1]: I looked into it and think it is too heavy for the temporary purpose of command line driver, which only needs to be told if the input is from a file or should it become interactive. I can get rid of getopt and let andb.exe become interactive if argv[1] is NULL, or treat argv[1] as input file, which will avoid the issue of command line argument processing.

**Pull request merged**